docs/components/rule.md

```diff
@@ -269,73 +269,16 @@ Flags:
                                  TLS CA to verify clients against. If no client
                                  CA is specified, there is no client
                                  verification on server side. (tls.NoClientCert)
-      --label=<name>="<value>" ...
```

Comment:
Let's review manually here, but @idoqo can you add an issue on Thanos for the task to print flags in sorted order. (Let's make sure to provide the use case).